composer/src/test/kotlin/com/gojuno/composer/ApkSpec.kt

```diff
@@ -0,0 +1,22 @@
+package com.gojuno.composer
+
+import org.assertj.core.api.Assertions.assertThat
+import org.jetbrains.spek.api.Spek
+import org.jetbrains.spek.api.dsl.context
+import org.jetbrains.spek.api.dsl.it
+
+class ApkSpec : Spek({
+
+    context("parse package from apk") {
+
+        val testApkPath by memoized { fileFromJarResources<InstrumentationSpec>("instrumentation-test.apk").absolutePath }
+
+        it("test runner is parsed correctly") {
```

Comment:
`it("parses test runner correctly")`, ingilish, do u speak it?

Reply:
runglish lol